product_kit_management/models/sale_order_line.py

```diff
@@ -0,0 +1,73 @@
+from odoo import Command, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    is_kit = fields.Boolean(string="Is Kit", related="product_id.is_kit", store=True)
```

Comment:
```suggestion
    is_kit = fields.Boolean(string="Is Kit", related='product_id.is_kit')
```
why storing a related field?

Reply:
My mistake, store=True should not be there. I misunderstood it with another field.🙃